### DISCUSSION TITLE
OpenAI新發表的語言模型ChatGPT

### DISCUSSION THREAD

**Discussion by brian431** [1101-GS3073-C/Python-Community-GS3073-C#498] [科技新知]
此模型善於模仿人類語言，也能回答大部分的問題。不過要注意的是，它經常給出錯誤的資訊，因為它其實不了解世界是怎麼運作的，而是靠閱讀大量文章，將各種資訊連結。不過作為娛樂用途還是挺有趣的，各位也可以試著玩玩看。
https://udn.com/news/story/122589/6810681

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by lx3211** [comment]
感覺距離可以跟機器人聊天的日子不遠了

**Comment by Sasaki1026** [comment]
但是叫他幫我寫code常常都是錯的

**Comment by xCk27x** [comment]
https://www.techbang.com/posts/102396-internet-celebrity-chatgpt-does-not-recognize-that-27-can-be
事實上chatGPT算數的能力仍不完整，openAI也有特別說chatGPT仍無法進行3位整數互乘，例如529*347這種就不行。

**Comment by williamyen0623** [comment]
會不會像某些科幻電影演的一樣，AI有了自我意識要消滅人類呢？一想到就覺得好恐怖，但以現階段來看，真是個相當方便且有趣的技術

**Comment by manipulateed** [comment]
這個api 能做到如此的技術，相比經歷很多很大量的資料訓練，才能成就他背後的模型，但他帶來方便性似乎降低人在自己思考的能力，而過度依賴這個模型

**Comment by NcuJerryKao** [comment]
感覺離人類滅亡的日子不遠了
霍金：人工智慧恐使人類滅絕

**Comment by limitx0** [comment]
感覺ChatGPT的中文不太好
用中文問問題或是問關於中文的問題他就不太會回答

**Comment by Yilin20021016** [comment]
這個東西用中文跟英文在速度上會差超級多 可能要再改改
